rest/models/quotes.go

```diff
@@ -19,9 +19,40 @@ type ListQuotesParams struct {
 	TimestampLTE *time.Time `query:"timestamp.lte"`
 	TimestampGT  *time.Time `query:"timestamp.gt"`
 	TimestampGTE *time.Time `query:"timestamp.gte"`
-	Order        *Order     `query:"order"`
-	Limit        *int       `query:"limit"`
-	Sort         *Sort      `query:"sort"`
+
+	Sort  *Sort  `query:"sort"`
+	Order *Order `query:"order"`
+	Limit *int   `query:"limit"`
+}
+
+func (p ListQuotesParams) WithTimestamp(c Comparator, q time.Time) *ListQuotesParams {
```

Comment:
intentionally choosing not to document these since the model itself should handle it